src/shared/counter-store.js

```diff
@@ -66,7 +66,11 @@ export default class CounterStore {
     this._config.push(numCounters);
 
     if (this._cache !== null) {
-      this._cache.push(NULL_NUMBER);
+      let cache = this._cache;
+
+      this._cache = new Uint32Array(this._namespaceCount);
```

Comment:
Which do we care about? It's IE10+. Older Node may have trouble, haven't fully investigated there.